.github/workflows/get-environment.yml

```
@@ -0,0 +1,321 @@
+
+                return true;
+              });
+
+              core.setOutput('target_stability', targetStability);
+            }
+
+            core.setOutput('is_targeting_feature_branch', isTargetingFeatureBranch);
+
+      - name: Get version from ${{ inputs.version_file }}
+        id: get_version
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2193:warning:1:36: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)